package.json

```diff
@@ -4,7 +4,7 @@
   "private": "private",
   "scripts": {
     "start": "node ./bin/www",
-    "test": "export NODE_ENV=test && mocha test"
+    "test": "NODE_ENV=test npm test"
```

Comment:
In this line you define what the command `npm test` do. If you write here `npm test` you are creating a loop and that's why your build is broken. Please replace back with `mocha`